plugins/woocommerce/client/admin/client/settings-payments/components/incentive-banner/incentive-banner.scss

```
@@ -4,11 +4,22 @@
 	box-shadow: none;
 	border-radius: 8px;
 
+	@media screen and ( max-width: 768px ) {
+		margin-left: -20px;
+		margin-right: -20px;
+		border-radius: 0;
+	}
+
+
 	&__content {
 		display: flex;
 		align-items: center;
 		padding: 0 calc(48px);
 
+		@media screen and ( max-width: 782px ) {
+			padding: 0 16px;
```

I recommend using the gap variables:
```suggestion
			padding: 0 $gap;
```

---

changed in [f2436e0](https://github.com/woocommerce/woocommerce/pull/54211/commits/f2436e036be6189a406e948419624a7ac95b1423) . thanks